src/tests/quadrature/test_gauss.f90

```diff
@@ -21,7 +21,8 @@ subroutine collect_gauss(testsuite)
             new_unittest("gauss-lobatto-analytic", test_gauss_lobatto_analytic), &
             new_unittest("gauss-lobatto-5", test_gauss_lobatto_5), &
             new_unittest("gauss-lobatto-32", test_gauss_lobatto_32), &
-            new_unittest("gauss-lobatto-64", test_gauss_lobatto_64) &
+            new_unittest("gauss-lobatto-64", test_gauss_lobatto_64), &
+            new_unittest("gauss-github-issue-619", test_fix_github_issue619) &
```

Comment:
Hi @milancurcic! I'll happily defer to your judgement and implement the requested change if you reject the following argument, but on first thought, I disagree. I don't think that "test-gauss-with-interval-provided" makes it clear why the test exists or what it was intended to fix. Having a reference to an actual tracked issue makes it clear that it exists in reaction to a bug that was actually shipped, and gives them a clear call to action in case they wish to understand the context around it's inclusion. It's a rather strange test without context.

Reply:
OK, that sounds good, thanks. Then I suggest including the full url to the GH issue in the docstring of the text.

Reply:
I added the link where I think you meant me to. Please let me know if I put it in the wrong place.